src/gui/qgsprojectionselectiontreewidget.cpp

```diff
@@ -947,17 +948,54 @@ void QgsProjectionSelectionTreeWidget::updateBoundsPreview()
                    .arg( rect.yMaximum(), 0, 'f', 2 );
   }
 
+  QStringList properties;
+  if ( currentCrs.isGeographic() )
+    properties << tr( "Geographic (uses latitude and longitude for coordinates)" );
+  else
+  {
+    properties << tr( "Units: %1" ).arg( QgsUnitTypes::toString( currentCrs.mapUnits() ) );
+  }
+  properties << ( currentCrs.isDynamic() ? tr( "Dynamic (relies on a datum which is not plate-fixed)" ) : tr( "Static (relies on a datum which is plate-fixed)" ) );
```

Comment:
I know this is me who wrote that dynamic = not plate-fixed, but I believe that, if that describes pretty much what exists now, we might get more complicated things in the future, like some of the new US datums being plate-fixed 'hiding' the plate motion but keeping a dynamic aspect for crustal deformation (like in California)... Anyway let's keep that "simple" description for now :-)

Reply:
Ok, I'll keep an eye out for any upstream wording changes and will adjust this string accordingly